Scripts/script-CrowdStrikeStreamingPreProcessing.yml

```diff
@@ -0,0 +1,38 @@
+commonfields:
+  id: CrowdStrikeStreamingPreProcessing
+  version: -1
+name: CrowdStrikeStreamingPreProcessing
+script: |-
+  import sys
+
+  res =False
+  for inc in demisto.incidents():
+      host = ''
+      for t in inc['labels']:
+          if t['type'] == 'System':
+              host = t['value']
+      sameIncidents = demisto.executeCommand("getIncidents",
+          {"query":"labels.value:\""+host+"\"  and labels.type:System"})
+      # if found sameIncidents found, add this incident data to war room
+      sameIncidentsCount = sameIncidents[0]['Contents']['total']
```

Comment:
Do you always have `total`? Just wondering if you'll have key error if for some reason you have an error. I think you should check for errors here.

Reply:
even if error.. not much I can do.. it's preprocessing nothing to return to user